src/main/java/hu/elte/eserial/annotation/EserialAnnotation.java

```diff
@@ -0,0 +1,27 @@
+package hu.elte.eserial.annotation;
+
+import java.lang.annotation.Annotation;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import hu.elte.eserial.annotation.enumeration.EserialAnnotationType;
+import hu.elte.eserial.util.AnnotationUtils;
+
+/**
+ * The parent annotation for all annotations influencing the serialization/deserialization process.
+ * Without having this on an annotation, it will not be considered.<br>
```

Comment:
What's the purpose of these `<br>`s?

Reply:
Here it's useless. Otherwise it's used for new lines.